voice/about.md

```diff
@@ -5,9 +5,10 @@
 
 ## REST API Reference Index
 
-| Resource                           | Description                                                           |
-|:-----------------------------------|:----------------------------------------------------------------------|
-| [`/calls`](methods/calls/about.md) | The Calls resource lets you make phone calls and modify active calls. |
+| Resource                                     | Description                                                           |
+|:---------------------------------------------|:----------------------------------------------------------------------|
+| [`/calls`](methods/calls/about.md)           | The Calls resource lets you make phone calls and modify active calls. |
+| [`/recordings`](methods/recordings/about.md) | The Recordings resource lets you retrieve information about recordings created during calls as well as manage recordings on a live call. |
```

Comment:
to manage recordings on a live call you need to use /calls/id/recordings right?

Reply:
That is true, but that endpoint is also present in the `recordings/about.md`.
There was a discussion about if that endpoint should be in the `calls/about.md` or in `recordings/about.md`. I didn't participate the discussion but right now this endpoint is present in both pages.